extensions/who-is-visiting-this-content/app.R

```diff
@@ -633,6 +895,24 @@ server <- function(input, output, session) {
 
     plotlyOutput("visit_timeline_plot", height = paste0(height_px, "px"))
   })
+
+  # Global UI elements ----
+
+  output$page_title_bar <- renderUI({
```

Comment:
Perhaps we can provide a URL / GUID somewhere in this title bar?

Reply:
I like this idea! I'll put the GUID there for now, as it seems like reasonable info to have while not being so long that it makes things difficult to lay out. (Plus, the Open in Connect button has the URL).

Reply:
![Screen Shot 2025-04-28 at 05 34 50 PM@2x](https://github.com/user-attachments/assets/989a309f-30a2-4a6b-856e-cf424c278310)